CHANGELOG-BETA.md

```diff
@@ -4,6 +4,26 @@ _Beta versions are released on the `beta` stream. The latest beta can be install
 
 For the latest stable changes, see [CHANGELOG.md](./CHANGELOG.md).
 
+## [4.0.27-beta]
+
+### Added
+
+- Throws error when passing both dev and alpha options for MySky (#257). by @amberm31 in https://github.com/SkynetLabs/skynet-js/pull/352
+- Add Dependabot Auto-Merge action by @m-cat in https://github.com/SkynetLabs/skynet-js/pull/413
+
+### Changed
+
+- Mysky login flow fixes by @m-cat in https://github.com/SkynetLabs/skynet-js/pull/403
```

Comment:
I like linking out to the user and PR. What tool did you use for this?

Reply:
I used github releases ui for that, unfortunately it doesn't handle -beta too well so I had to copy only the relevant changelog because it generated it since 3.0.2 or sth